### PR TITLE
cardano tokens list update

### PR DESCRIPTION
**Pull request by pavanvora**
### 📝 Description

This PR includes the updated list of supported tokens by ledger nano APP version 6.0.2. It has a dependency on this https://github.com/LedgerHQ/ledger-live/pull/2756 PR

### ❓ Context

- **Impacted projects**: `ledgerjs` `lld` `llc`
- **Linked resource(s)**: `` 

### ✅ Checklist

- [x] **Test coverage** 
- [ ] **Atomic delivery**
- [x] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: libs/ledgerjs/packages/cryptoassets/src/data/cardanoNative.json [hunk 1]
```diff
@@ -37,7 +37,7 @@
     "SUNDAE",
     6,
     false,
-    false
+    true
```
Comment (haammar-ledger) [inline libs/ledgerjs/packages/cryptoassets/src/data/cardanoNative.json:40]: This sets `disableCountervalue` to `true`, is this expected? (same question for other tokens too)

Comment (pavanvora) [reply libs/ledgerjs/packages/cryptoassets/src/data/cardanoNative.json:40]: no it is not, I wasn't sure before so it was set to true for all. I'll update it

**Review by haammar-ledger — COMMENTED** [review]
The changes look ok, thanks :+1: 

You just want to update some tests' snapshots, you can do so with:
```
pnpm common jest --runTestsByPath src/currencies/sortByMarketcap.test.ts -u
pnpm desktop test:jest --runTestsByPath src/generate-cryptoassets-md.test.js -u
```

**Comment by haammar-ledger** [comment]
Hi @pavanvora could you please rebase on develop once again :pray: thank you.

**Review by haammar-ledger — APPROVED** [review]
@pavanvora Any progress on this PR? Just need to rebase on develop until CI is green AFAICT...

**Comment by pavanvora** [comment]
Hi @haammar-ledger, I completely missed to update this PR 🙏 . Just pushed the changes.

**Pull request closed without merging**